File: server/WebApplication2/Models/Review.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication2.Models
{
    public class Review
    {
        [Key]
        public int id { get; set; }
        public string title { get; set; }
        public string description { get; set; }
        public int rating { get; set; }
        public int album_id { get; set; }
        public virtual Album album { get; set; }
        public int user_id { get; set; }
        public virtual User user { get; set; }
    }
}

File: client/src/app/Model/Review.ts
import { Album } from "./Album";
import { User } from "./User";

export class Review {
    id: number = 0;
    title: string =  "";
    description: string = ""; 
    rating: number = 0;
    album_id: number = 0
    album: Album = new Album;
    user_id: number = 0;
    user: User = new User;
}